src/app/collective-rewards/rewards/builders/EstimatedRewards.tsx

```diff
@@ -47,22 +46,33 @@ const TokenRewards: FC<TokenRewardsProps> = ({ builder, gauge, token: { id, symb
     data: totalPotentialRewards,
     isLoading: totalPotentialRewardsLoading,
     error: totalPotentialRewardsError,
-  } = useGetTotalPotentialReward()
+  } = useReadBackersManager({
+    functionName: 'totalPotentialReward',
+  })
   const {
     data: rewardShares,
     isLoading: rewardSharesLoading,
     error: rewardSharesError,
-  } = useGetRewardShares(gauge)
+  } = useReadGauges({ addresses: [gauge], functionName: 'rewardShares' })
```

Comment:
Should we have a `useReadGauge` where we need to call a function of a single gauge?

Reply:
I considered it, but there aren't enough instances of this to justify it, I decided. This approach works just as well for a single use case.

Reply:
Creating a new hook will make this and other usages a bit clearer

Reply:
done in https://github.com/RootstockCollective/dao-frontend/pull/926/commits/ca312077ed6d20ce6004e00d8c25fea60477e763